Symfony/CS/Fixer/Symfony/MethodSeparationFixer.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use SplFileInfo;
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * Make sure there is one blank line above and below a method.
+ */
+final class MethodSeparationFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAnyTokenKindsFound(Token::getClassyTokenKinds());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(SplFileInfo $file, Tokens $tokens)
+    {
+        $types = array(array(T_CLASS), array(T_INTERFACE));
+        if (defined('T_TRAIT')) {
+            $types[] = array(T_TRAIT);
+        }
+
+        $index = $tokens->getSize() - 1;
+        do {
```

Comment:
I don't think it will be less, cause I would need to add a `getControlTokens` method.
Besides, I don't like calling functions/methods like;
`for ($index = $tokens->count() - 1; 0 <= $index; --$index) {`
I'd rather fetch the count once and work with that ;)